File: noPaperService_common/Entities/CertComponent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography.X509Certificates;
using System.Text;
using System.Threading.Tasks;

namespace noPaperService_common.Entities
{
    public class CertComponent
    {
        public X509Certificate2 cert = null;
        public String thumbprint;
        public String valid;
        public String issued;
        public String fullIssued;
        public String organization;
        public String subject;
        public String fullSubject;
        public String serialNumber;

        public CertComponent(X509Certificate2 cert)
        {
            this.cert = cert;
        }
    }
}

File: noPaperService_Api/Models/CertComponent.vb
Namespace Models
    Public Class CertComponent
        Public Property cert As Security.Cryptography.X509Certificates.X509Certificate2 = Nothing
        Public Property thumbprint As String = Nothing
        Public Property valid As String = Nothing
        Public Property issued As String = Nothing
        Public Property fullIssued As String = Nothing
        Public Property organization As String = Nothing
        Public Property subject As String = Nothing
        Public Property fullSubject As String = Nothing
        Public Property serialNumber As String = Nothing
        Public Property subjectPost As String = Nothing
    End Class
End Namespace